chainguard-dev/edu
# Add policy controller guide on verifying Chainguard images

### What should this PR do?
Add a guide with embedded terminal showing how to use Policy Controller to only admit keyless signed Chainguard images into a cluster.

### Why are we making this change?
Part of the ongoing series about policy controller tutorials.

### What are the acceptance criteria? 
The tutorial should be live under `/open-source/sigstore/policy-controller/using-policy-controller-to-verify-signed-chainguard-images/` on Netlify preview.

### How should this PR be tested?
Test the steps locally using http://localhost:1313/open-source/sigstore/policy-controller/using-policy-controller-to-verify-signed-chainguard-images/
